src/PhpSpec/Runner/Maintainer/CollaboratorsMaintainer.php

```diff
@@ -140,6 +130,9 @@ private function generateCollaborators(CollaboratorManager $collaborators, \Refl
                 if (($indexedClass = $this->getParameterTypeFromIndex($classRefl, $parameter))
                     || ($indexedClass = $this->getParameterTypeFromReflection($parameter))) {
                     $collaborator->beADoubleOf($indexedClass);
+                    if (null !== $indexedClass) {
```

Comment:
yes. As i said, just a small off topic note.
I've removed the check. The test is - obviously - still green.